app/models/family.rb

```diff
@@ -58,6 +58,9 @@ class Family
   belongs_to :person, optional: true
   has_many :hbx_enrollments
 
+  # An application can be individual_market_application, financial_assistance_application, or shop_market_application
+  has_many :applications, class_name: 'Sbm::Application'
+
   # Collection of insured:  employees, consumers, residents
 
```

Comment:
add method to pull latest determined application of any type. also will likely need to pull the type of latest determined